### PR TITLE
Enhancement Community Member Profile Social Links #4367

### LINKED ISSUE #4367: Enhancement Community Member Profile Social Links
### Desired Behavior

Add Meshery Cloud public profile link of the member next to LinkedIn icon.

1. Add Meshery Cloud logo next to the LinkedIn icon.
2. Add meshery cloud UUID as input in "member-profile-template".

https://layer5.io/community/members/aadhitya-amarendiran

### Screenshots / Mockups

Screenshot 2023-06-09 at 2 32 30 PM

### Acceptance Tests

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Manish254252**
**Description**

This PR fixes #4367 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 063153149d1227478ae00994182b5571801bb091 at: https://64905cdfbb4a666d3746aeea--layer5.netlify.app

File: src/sections/Community/Member-single/index.js [hunk 1]
```diff
@@ -274,6 +274,13 @@ const MemberSingle = ({ frontmatter }) => {
                       </Link>
                     </li>
                   )}
+                  {
+                    <li>
+                      <a href="https://layer5.io/cloud-native-management/meshery" target="_blank" rel="noreferrer">
+                        <img src={mesheryLogo} alt="meshery-icon"></img>
```
Comment (HarshDobariya79) [inline src/sections/Community/Member-single/index.js:280]: Nice work @Manish254252. Can we add the white version of the logo here to make things consistent? The rest looks good to me. Thank you @Manish254252.

**Review by Mohith234 — CHANGES_REQUESTED** [review]
- Add meshery cloud UUID as input in "member-profile-template" as mentioned in the 2nd point of the issue. @Manish254252 
- Also do change the logo to a white version as mentioned by @HarshDobariya79

File: src/sections/Home/Partners-home/index.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ const settings = {
   responsive: [
     {
       breakpoint: 1400,
-      settings: "unslick"
+      settings: "slick"
```
Comment (senali-d) [inline src/sections/Home/Partners-home/index.js:24]: @Manish254252 What is the purpose of this change?

**Review by senali-d — CHANGES_REQUESTED** [review]
Show Meshery logo only when there is UUID for the member profile.

**Comment by vishalvivekm** [comment]
@Manish254252 Please incorporate the changes suggested above.

File: src/sections/Community/Member-single/index.js (hunk 1 above)
Comment (Ghat0tkach) [reply src/sections/Community/Member-single/index.js:280]: this is adding meshery-cloud to every member profile , we wish to add only on users who are present in meshery cloud

**Comment by Ghat0tkach** [comment]
> @Manish254252 Please incorporate the changes suggested above.
hey @vishalvivekm 
This PR is adding meshery-cloud-logo on every profile , You remember the freedisch's meshery cloud id you gave me? I had tough time figuring it out , However after seeing this issue and PR  . I was able to implement the meshery-cloud in profile-badges both in index.md as well as single-member.js
It's working same as templates 🙌

File: src/sections/Community/Member-single/index.js (hunk 1 above)
Comment (vishalvivekm) [reply src/sections/Community/Member-single/index.js:280]: right @Ghat0tkach

File: src/sections/Home/Partners-home/index.js (hunk 2 above)
Comment (vishalvivekm) [reply src/sections/Home/Partners-home/index.js:24]: @Manish254252

**Comment by vishalvivekm** [comment]
@Ghat0tkach - 
replace "the last portion of ....." with
: `<user ID> only UUID https://meshery.layer5.io/user/<uuid>`

and would you like to take this PR forward from here as we didn't hear from @Manish254252 ?

**Comment by Ghat0tkach** [comment]
Thanks a lot! Will Sure 
@vishalvivekm  , I have already maded changes in a branch shall I incorporate the same changes? 
you can check the changes I made [here](https://github.com/layer5io/layer5/compare/master...Ghat0tkach:layer5:Meshery-cloud-badge)

**Comment by vishalvivekm** [comment]
and yeah to your point we only want the meshery-cloud on only those members' profiles who do have a meshery-cloud account ( which they for sure would in most of the cases).

**Comment by Ghat0tkach** [comment]
> and yeah to your point we only want the meshery-cloud on only those members' profiles who do have a meshery-cloud account ( which they for sure would in most of the cases).

Thanks a lot ! Just a little doubt @vishalvivekm . Apologies
Shall I make changes in this PR or raise a new PR(as its already ready with the changes + tested functionality)

**Comment by vishalvivekm** [comment]
> Thanks a lot! Will Sure @vishalvivekm , I have already maded changes in a branch shall I incorporate the same changes? There are couple of more changes we need to do in order to execute it you can check the changes I made [here](https://github.com/layer5io/layer5/compare/master...Ghat0tkach:layer5:Meshery-cloud-badge)

all yours :)

**Comment by vishalvivekm** [comment]
> > and yeah to your point we only want the meshery-cloud on only those members' profiles who do have a meshery-cloud account ( which they for sure would in most of the cases).
> 
> Thanks a lot ! Just a little doubt @vishalvivekm . Apologies Shall I make changes in this PR itself or raise a new PR(as mine is already ready with the changes + tested functionality)

howsoever you deem fit @Ghat0tkach

**Comment by Ghat0tkach** [comment]
@vishalvivekm  , Thanks a lot!!
